website/templates/deviceactions.html

```diff
@@ -0,0 +1,123 @@
+{% extends "base.html" %}
+{% block page_content %}
+    <h1> Motion Sensor Action Settings </h1>
+    <br></br>
+    <hr></hr>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet"/>
+    <div class="container">
+
+        <form class="form-horizontal" role="form">
```

Comment:
Don't forget to use syntax checking in PyCharm to make sure you close all tags (which is not the case for e.g. this form).